ee/vellum_ee/workflows/display/utils/expressions.py

```diff
@@ -275,16 +276,13 @@ def serialize_value(executable_id: UUID, display_context: "WorkflowDisplayContex
         return serialize_value(executable_id, display_context, child_descriptor)
 
     if isinstance(value, WorkflowInputReference):
-        try:
-            workflow_input_display = display_context.global_workflow_input_displays[value]
-        except KeyError:
-            inputs_class_name = value.inputs_class.__name__
-            workflow_class_name = display_context.workflow_display_class.infer_workflow_class().__name__
-            raise UnsupportedSerializationException(
```

Comment:
we can still raise this exception if `value.inputs_class != display_context.workflow_display_class.infer_workflow_class().get_inputs_class()`